doc/sailfish-webview-webview.qdoc

```diff
@@ -14,4 +14,20 @@
   import namespace.
 */
 
+/*!
+  \qmlproperty PopupProvider WebView::popupProvider
+  \brief A provider of custom user interaction components
+
+  Client applications can define their own custom user interaction components
+  which extend the various interface types defined in the
+  \l{Sailfish.WebView.Popups} namespace.
+
+  In order for those custom components to be used by the WebView, the client
+  must register a \c{PopupProvider} instance (which specifies the various
```

Comment:
```suggestion
  must register a \l{PopupProvider} instance (which specifies the various
```

Reply:
Actually \l{PopupProvider} could be simplified to PopupProvider

Reply:
done